### ISSUE TITLE
blastp: choosing incorrect defaults

### ISSUE THREAD

**Issue by aurreco-uga** [VEuPathDB/service-multi-blast#129]
from error emails
B6BFA154DFD985F41240B6CF5E156DE3

when the user changes the matrix to PAM250, we dont select correct defaults for -gapopen":11,"-gapextend":1,
it ends in 500, **error is clear to the user**

LOGS
A secondary issue is that the logs show 200, all good????? (check with ellie)

VALIDATOR
2022/04/03 14:13:27 Input JSON:  {"-outfmt":{"type":15},"-max_target_seqs":100,"-word_size":6,"-gapopen":11,"-gapextend":1,"-matrix":"PAM250"}

**Comment by Foxcapades** [comment]
The config in the DB is:

```json
{
  "tool": "blastp",
  "-outfmt": {
    "type": 15
  },
  "-max_target_seqs": 100,
  "-db": "//db/PlasmoDB/build-57/Pyoeliiyoelii17X/blast/Pyoeliiyoelii17XAnnotatedProteins",
  "-word_size": 6,
  "-gapopen": 11,
  "-gapextend": 1,
  "-matrix": "PAM250"
}
```

This means either there is a bug in the translation of the input format to the internal format or there is a client issue.

**Comment by aurreco-uga** [comment]
yes this might be a client issue.. with previously stored param values...
<img width="1192" alt="Screen Shot 2022-04-05 at 5 09 09 PM" src="https://user-images.githubusercontent.com/16354313/161850506-2d61ed6d-3931-4a84-9275-af99f4def88e.png">
